source/wifi/wifi_hal.c

```diff
@@ -1767,8 +1767,25 @@ INT wifi_getRadioOperatingChannelBandwidth(INT radioIndex, CHAR *output_string)
 {
     if (NULL == output_string)
         return RETURN_ERR;
+    WIFI_ENTRY_EXIT_DEBUG("Inside %s:%d\n",__func__, __LINE__);
+    char cmd[1024] = {0}, buf[64] = {0};
+    char interface_name[50] = {0};
+    int ret = 0, len=0;
 
-    snprintf(output_string, 64, (radioIndex == 0) ? "20MHz" : "40MHz");
+    snprintf(cmd, sizeof(cmd),"iw dev %s%d info | grep 'width' | cut -d  ' ' -f6",AP_PREFIX, radioIndex);
+    ret = _syscmd(cmd, buf, sizeof(buf));
+    printf("the ret is %d \n",ret);
```

Comment:
Please use wifi_dbg_printf instead of printf.
You can add also __func__ and __line__ for more verbose information